packages/realm-react/src/RealmProvider.tsx

```diff
@@ -41,43 +41,71 @@ type ProviderProps = PartialRealmConfiguration & {
    * instance outside of a component that uses the Realm hooks.
    */
   realmRef?: React.MutableRefObject<Realm | null>;
+  /**
+   * The fallback component to render if the Realm is not open.
+   */
+  fallback?: React.ComponentType<unknown> | React.ReactElement | null | undefined;
   children: React.ReactNode;
-};
+} & PartialRealmConfiguration;
+
+/**
+  Represents the provider returned from using an existing realm at context creation i.e. `createRealmContext(new Realm))`.
+ */
+export type RealmProviderFromRealmFC = React.FC<Pick<RealmProviderProps, "children">>;
```

Comment:
To me `FC` here feels a bit like an implementation detail.
```suggestion
export type RealmProviderFromRealm = React.FC<Pick<RealmProviderProps, "children">>;
```